docs/release-notes/cloud/cloud-2023-10.md

```diff
@@ -30,6 +30,13 @@ These release notes are for the Codacy Cloud updates during October 2023.
 -   TSQLLint can now also detect and analyze files with an `sql` extension. (TS-543)
 -   Codacy no longer supports and now skips the analysis of branches named `HEAD` or matching the pattern `refs/heads/*`, as these are Git reserved terms. (IO-830)
 
+## Feature removal
+
+Due to their limited adoption, we removed the following options for the issues listed on the **Issues** page:
```

Comment:
```suggestion
Due to their limited adoption, we removed the following options from the options menu of the issues listed on the **Issues** page:
```

Reply:
Tweaking your sentence to avoid "options" repetition (consistent with [this sentence](https://github.com/codacy/docs/blob/master/docs/repositories/issues.md?plain=1#L56)).

```suggestion
Due to their limited adoption, we removed the following options from the menu of each issue listed on the **Issues** page:
```